### PR TITLE
Bayonet attachable | Gun melee fix

### PR DESCRIPTION
**Pull request by slayerplayer99**
my god im out of it. okay this should be the one

### CODE REVIEW

File: code/modules/marines/Marine_Weapons_V2.dm [hunk 1]
```diff
@@ -189,6 +189,8 @@
 	ejectshell = 0 //Caseless
 	fire_delay = 4
 	slot_flags = SLOT_BACK
+	var
```
Comment (rahlzel) [inline code/modules/marines/Marine_Weapons_V2.dm:192]: Why are you changing the coding style? Why is this var defined over the span of two lines, and with an empty comment?

File: code/modules/marines/Marine_Weapons_V2.dm [hunk 2]
```diff
@@ -196,7 +198,19 @@
 		update_icon()
 		return
 
-
+	attackby(var/obj/item/A as obj, mob/user as mob)
+		if(istype(A,/obj/item/weapon/attachable))
+			if(customized==0)
+				force+=A.force
+				src.contents+=A
+				customized+=1
```
Comment (rahlzel) [inline code/modules/marines/Marine_Weapons_V2.dm:206]: If `customized` can only be 1 or 0, why is this +=?

File: code/modules/marines/Marine_Weapons_V2.dm [hunk 3]
```diff
@@ -210,8 +224,13 @@
 		..()
 		if(empty_mag)
 			icon_state = "m41a"
+			if(customized>=1)                  //
+				icon_state = "m41ac1"
+
 		else
 			icon_state = "m41a0"
+			if(customized>=1)                //
```
Comment (rahlzel) [inline code/modules/marines/Marine_Weapons_V2.dm:232]: Can these two ever be greater than 1?

File: code/modules/marines/Marine_Weapons_V2.dm [hunk 4]
```diff
@@ -232,6 +251,24 @@
...
+			src.contents-=A
+			usr<<"<font color=teal>You remove [A.name] from your rifle."
+			customized-=1
+			if(customized<=0)
+				customized=0
+		else
+			usr<<"You don't have an customizable attachment to your rifle."
+			return
+			*/
+/*
```
Comment (rahlzel) [inline code/modules/marines/Marine_Weapons_V2.dm:271]: Why was this verb commented out? Seems like something useful for players to be able to do: remove an attachment.

**Comment by slayerplayer99** [comment]
Customized +1 cause maybe we would want to add another attachment for future purposes and only needing to change the if checks. Commented out the verb because it's expiremental for now until I get it right and joshuu needs to add in the rest that is missing and that var indent is just a habit I sometimes do

**Comment by rahlzel** [comment]
Please stick to the coding style. A few extra seconds makes things easier to read and can save time if something looks like it might be broken because it's formatted differently.

This goes for defining variables on two lines and spacing in between operators (+=, =, etc)

**Pull request closed without merging**